### DISCUSSION TITLE
CDL 2008-2010 issues for Farmington, NM area

### DISCUSSION THREAD

**Discussion by cgmorton** [Open-ET/crop-type-assets#4] [General]
The CDL values for 2008-2010 in northwest New Mexico in the Farmington area do not match up well with what we see in Landsat (and NAIP 2009) imagery.  The images below show the CDL working back from 2011 to 2008.

We are tentatively proposing that we use the 2011 CDL value for 2008-2010 for this region.  The exact region or fields still needs to be defined, but should be similar to the extent in the screenshots below.

2011 CDL
<img width="1163" alt="Screenshot 2023-12-04 at 11 27 40 AM" src="https://github.com/Open-ET/crop-type-assets/assets/9002566/1049cf67-7097-44e7-982c-790c3ab97551">

2010 CDL
<img width="1169" alt="Screenshot 2023-12-04 at 11 28 40 AM" src="https://github.com/Open-ET/crop-type-assets/assets/9002566/9ea3f214-ac93-49f2-9c84-0aa86c64ab7b">

2009 CDL
<img width="1171" alt="Screenshot 2023-12-04 at 11 28 53 AM" src="https://github.com/Open-ET/crop-type-assets/assets/9002566/7d7291d3-829e-443a-bfcb-838b4e2c75fb">

2008 CDL
<img width="1168" alt="Screenshot 2023-12-04 at 11 29 02 AM" src="https://github.com/Open-ET/crop-type-assets/assets/9002566/d016f3f6-8156-447c-bbca-e5d81c31ed02">

**Comment by cpearson1** [comment]
![image](https://github.com/Open-ET/crop-type-assets/assets/14983120/3103ef63-0a60-4f06-8b03-c7631df1f7e4)

**Reply by cpearson1** [reply]
The majority of Farmington, NM fields from 2008-2010 are misclassified as Shrubland (CDL 152). A large step change decrease in 152 acreage is seen in 2011.

**Comment by cpearson1** [comment]
![image](https://github.com/Open-ET/crop-type-assets/assets/14983120/b73bd65a-8162-4ca4-a0ce-a4f1ef48a860)

**Reply by cpearson1** [reply]
Summertime NDVI also shows larger ag extent than observed in CDL for 2008-2010. https://climengine.page.link/vZ51

**Reply by cpearson1** [reply]
Google Earth Timelapse shows that most center pivot development occurs in early 2000s.
https://earthengine.google.com/timelapse#v=36.54165,-108.12795,8.313,latLng&t=1.73&ps=50&bt=19840101&et=20221231
